syntax = "proto3";

package chord;

service ChordService {
    rpc SendHello (HelloRequest) returns (HelloReply);
    rpc AddNumbers (AddRequest) returns (AddReply);
}

message HelloRequest {
    string message = 1;
}

message HelloReply {
    string reply = 1;
}

message AddRequest {
    int32 number1 = 1;
    int32 number2 = 2;
}

message AddReply {
    int32 result = 1;
}
